bucket/gpg4win-portable.json

```
@@ -0,0 +1,42 @@
+{
+    "version": "3.1.10",
+    "homepage": "https://www.gpg4win.org",
+    "description": "GNU Privacy Guard for Windows is encryption software for files and emails.",
+    "license": "GPL-2.0-or-later",
+    "url": "https://files.gpg4win.org/gpg4win-3.1.10.exe#/dl.7z",
+    "hash": "7f3ceaf54bcff0f63716900f6769b93597aae33e188e88792aa484a93d056dc9",
+    "depends": "gpg",
+    "env_add_path": "bin",
+    "shortcuts": [
+        [
+            "bin\\gpa.exe",
+            "GPA"
+        ],
+        [
+            "bin\\Kleopatra.exe",
+            "Kleopatra"
+        ]
+    ],
+    "persist": "home",
+    "post_install": [
+        "Remove-Item \"$dir\\`$*\", \"$dir\\gpg4win-uninstall.exe.nsis\" -Force -Recurse",
+        "$kdeconf = @(",
+        "    '[KDE]'",
+        "    'KDEHOME=home/kleopatra'",
+        "    '[XDG]'",
+        "    'XDG_DATA_HOME=home/kleopatra'",
+        "    'XDG_CONFIG_HOME=home/kleopatra'",
+        ")",
```

- UTF8 is with BOM in PW5
- Is force needed?

```suggestion
        ") | Set-Content \"$dir\\bin\\kde.conf\" -Encoding ASCII"
```